invenio_checks/base.py

```diff
@@ -67,9 +67,6 @@ def load_from_entry_points(self, app, ep_name):
         """Load checks from entry points."""
         for ep in set(importlib_metadata.entry_points(group=ep_name)):
             check_cls_or_func = ep.load()
-            if callable(check_cls_or_func):
-                check_cls = check_cls_or_func(app)
-            else:
-                check_cls = check_cls_or_func
+            check_cls = check_cls_or_func
```

Comment:
minor: I should have copy-pasted directly from the logic in [`invenio-requests`](https://github.com/inveniosoftware/invenio-requests/blob/b6630684fe2e15cff26c1114accfb96b703fd051/invenio_requests/ext.py#L122-L124). Classes are callable too apparently 😅 

This is overall useful, because there are cases where we want to have dynamic approaches to modify/select the underlying class (e.g. based on some config). It's not something we currently need, but always handy to come across in the future (and a common pattern with other types of entrypoints too)